src/framework/global/io/dir.h

```diff
@@ -33,7 +33,7 @@ namespace mu::io {
 class IFileSystem;
 class Dir
 {
-    INJECT_STATIC(IFileSystem, fileSystem)
+    static Inject<IFileSystem> fileSystem;
```

Comment:
Could do `static inline`. That should remove the need to put something in cpp file.